.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Closes <!-- issue reference -->
+
+## Type of change
+
+-   [ ] New feature (non-breaking change which adds functionality)
+-   [ ] Bug fix (non-breaking change which fixes a bug)
+-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+-   [ ] Refactoring (only inner code structure was changed)
+-   [ ] Documentation update (no code change was made)
+-   [ ] CI/CD
+-   [ ] Miscellaneous
+
+## What I have made
+
+<!-- write down what changes have been made, what was removed/added -->
+<!-- e.g. I added a new route called x, it has the parameters y and z -->
+
+## Checklist
+
+Either tick or cross out the items that do not apply (using \~\~example text\~\~) and give a reason why the item does not apply.
+
+-   [ ] I have added tests that prove my fix is effective or that my feature works
+    -   [ ] unit tests
+    -   [ ] integration tests
+    -   [ ] end-to-end tests
+-   [ ] I have commented my code, particularly in hard-to-understand areas
```

Comment:
```suggestion
- [ ] I have followed the coding guidelines and style
- [ ] I have updated the documentation (README.md, API documentation, general documentation) and commented my code 
- [ ] I have added tests that prove my fix is effective or that my feature works
    -   [ ] unit tests
    -   [ ] integration tests
    -   [ ] end-to-end tests
 - [ ] (for reviewer): I have checked the implemented functionality against the requirements
```

Reply:
This was due to the prettier config, I changed it.

Reply:
Not only changed the indentation, but added other cases. What to you think of them?